src/movies.js

```diff
@@ -1,12 +1,65 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+const getAllDirectors = movies.map(function(movie){
+    return movie.director
+})
+
+console.log(getAllDirectors)
+
+
+
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
+// const getAllDirectors = movies.map(function(movie){
+//     return !movies.includes(movie);
+// })
+
+// console.log(getAllDirectors)
+
+
+
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
 
+
+
+// function howManyMovies(arr)
+
+
+const howManyMovies = movies.filter(function(movie) {
+    if movie.director === "Steven Spielberg" && if movie.genre.includes "Drama"
+})
+     
+
+
+
+
+    // const placesWithPool = places.filter(function (place) {
+    //     return place.pool === 'yes';
+    // }).map(function (place) {
+    //     return {
+    //         title: place.title
+    //     }
+    // });
+
```

Comment:
the start of this is pretty accurate. You don't need the second if though and it would have worked: 

```
function howManyMovies(arr) {
  let spielbergMovies = arr.filter(function(movie) {
    if (
      movie.director === "Steven Spielberg" &&
      movie.genre.includes("Drama")
    ) {
      return true;
    }
  });
  return spielbergMovies.length;
}
```